### ISSUE TITLE
Not an issue but a Question on deleting a post?

### ISSUE THREAD

**Issue by Therealjosephchrzempiec** [PeterBohai/social-network#1]
Hello, Can we delete a post or a commant we made?

**Comment by PeterBohai** [comment]
I don't believe so as this project was made specifically for one of the Harvard CS50 course projects as specified here:
https://cs50.harvard.edu/web/2020/projects/4/network/

Deletion of posts or comments were not part of the 2020 requirements. Not sure if that is a requirement for current or future course versions.
